src/datasets/utils/hub.py

```diff
@@ -6,7 +6,7 @@
 
 
 def hf_hub_url(repo_id: str, path: str, revision: Optional[str] = None) -> str:
-    if version.parse(hfh.__version__) < version.parse("0.11.0"):
+    if version.parse(hfh.__version__) < version.parse("0.11.0.rc0"):
```

Comment:
I'm not sure it makes sense to have in our code a reference to a release-candidate version...